<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimplePrompter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="6">
     <widget class="ScriptView" name="scriptView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Rundown:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="sectionCombo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="settingsButton">
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="icon">
       <iconset resource="simpleprompter.qrc">
        <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Section:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="GroupedComboBox" name="rundownCombo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QToolButton" name="reloadButton">
      <property name="text">
       <string>Reload</string>
      </property>
      <property name="icon">
       <iconset resource="simpleprompter.qrc">
        <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GroupedComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">groupedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ScriptView</class>
   <extends>QWidget</extends>
   <header location="global">scriptview.h</header>
   <container>1</container>
   <slots>
    <slot>forward()</slot>
    <slot>back()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="simpleprompter.qrc"/>
 </resources>
 <connections/>
</ui>
